csuntechlab/iSTART
# STR-26 -- FE: Create Generic Dashboard layout for use by everyone / Comparison and Intervention

# Description
Created a dashboard component that will be used to display content modules and alerts
  
# Testing
(no need to run jest)
1. login
2. visit /interventionGroup and /comparisonGroup routes
3. Make sure that navbar and notification are aligned properly
4. NOTE: Thumbnails for content modules were not made as that is covered on STR-28
5.make sure CSS properties are used correctly and fitting of bem standards
6. rejoice and live life to the fullest
